extensions/amp-analytics/0.1/analytics-group.js

```diff
@@ -70,12 +84,31 @@ export class AnalyticsGroup {
       eventType,
       this.root_.getType()
     );
-    const unlisten = tracker.add(
-      this.analyticsElement_,
-      eventType,
-      config,
-      handler
-    );
-    this.listeners_.push(unlisten);
+    let unlisten;
+    const deferred = new Deferred();
+    const task = () => {
+      unlisten = tracker.add(
+        this.analyticsElement_,
+        eventType,
+        config,
+        handler
+      );
+      this.listeners_.push(unlisten);
+      deferred.resolve();
+    };
+    if (
+      this.triggerCount_ < IMMEDIATE_TRIGGER_THRES ||
+      !isExperimentOn(this.win_, 'analytics-chunks')
+    ) {
```

Comment:
Probably worth it to leave a comment about why we want to execute the first `IMMEDIATE_TRIGGER_THRES` triggers immediately (aka increase active pageview ping).

Reply:
Added comment